data_processing/eia.py

```diff
@@ -0,0 +1,2999 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+
+# Commenting out the path for the actual data
+
+# path = 'C:\\Users\\Atwater\\research\\data\\eia_generation_'
+
+path = '.\\tests\\eia_sample_'
```

Comment:
```suggestion
path = './tests/eia_sample_'
```

The way the path is written right now works on windows, but not on Linux. This suggestion fixes the issue.

Reply:
Actually, making this a global variable breaks the functions because the only file it knows about is the one in ``tests``. 

Instead, tell people where they should download the data and set path

``path = "../../data/``

Up one directory to ``pride``, then up one more to ``research`` then go to ``data``.